grammar.js

```diff
@@ -80,10 +77,16 @@ module.exports = grammar({
       '{}',
       /\{\{+[0-9]*/,
       '(',
+      ')',
+      '[',
+      ']',
+      '[\'',
+      '\']',
```

Comment:
Are `['` and `']` important special cases or could we drop these?

Reply:
I think that was done to work around the b:sh_indent_options mentioned earlier. So if those are fixed by your other changes (or deemed "won't fix" by simply wrapping them in backticks) they could indeed go.